documentation/book/proc-deploying-kafka-cluster.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-cluster.adoc
+
+[id='deploying-kafka-cluster-{context}']
+= Deploying the Kafka cluster
+
+You can deploy an ephemeral or persistent Kafka cluster to {ProductPlatformName} on the command line.
+ifdef::OpenShift[]
+You can also deploy clusters in the {OpenShiftName} console.
```

Comment:
How exactly do we expect tis to work? Through the templates? Through creating the YAML? I wonder if we should either remove it (templates are being deprioritized AFAIK) or make it generic (most Kube clusters have some dashboard where you can create an YAML)